src/Sylius/Bundle/CoreBundle/spec/Doctrine/ORM/UserRepositorySpec.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\Doctrine\ORM;
+
+use Doctrine\ORM\AbstractQuery;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Query;
+use Doctrine\ORM\Query\Expr;
+use Doctrine\ORM\Query\FilterCollection;
+use Doctrine\ORM\QueryBuilder;
+use Pagerfanta\Pagerfanta;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Core\Repository\UserRepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class UserRepositorySpec extends ObjectBehavior
```

Comment:
IMO specifying a repository does not make sense, it is based on external collaborators and has no own logic.